.github/workflows/build-server.yml

```diff
@@ -1,16 +1,26 @@
 name: Build+Test Server
 
-on: [ push, pull_request ]
+on:
+  push:
+    paths:
+      - "server/**/*"
+  pull_request:
+    paths:
+      - "server/**/*"
```

Comment:
very cool, I didn't know this was possible

Reply:
Yea, I just learn about it maybe a month back